lib/Dialect/FIRRTL/Import/FIRTokenKinds.def

```diff
@@ -127,6 +127,7 @@ TOK_KEYWORD(of)
 TOK_KEYWORD(old)
 TOK_KEYWORD(output)
 TOK_KEYWORD(parameter)
+TOK_KEYWORD(Path)
```

Comment:
Currently this is alpha-sorted case-sensitive, please either change that or put this up with the capitals!